mitmproxy/contentviews/__init__.py

```diff
@@ -202,8 +202,9 @@ def get_content_view(
         desc, content = ret
         error = None
     # Third-party viewers can fail in unexpected ways...
-    except Exception:
-        desc = "Couldn't parse: falling back to Raw"
+    except Exception as e:
+        desc = str(e) + ": Couldn't parse: Falling back to Raw"
+        traceback.print_exc()
```

Comment:
´The changes here in this file are unrelated to the bugfix, please remove them.

(Without going into too much detail: The existing API contracts are weird here, and a return value of None does not only indicate no view. But that needs to be fixed properly.)